### ISSUE TITLE
Building TextExtraction

### ISSUE THREAD

**Issue by cvalde** [KevM/tikaondotnet#121]
Hello, I´m trying to build TikaOnDotnet.TextExtraction myself to be able to debug it and learn a bit. However, I get
'C:\projects\tikaondotnet-master\src\TikaOnDotnet.TextExtractor\..\..\SolutionInfo.cs' not found.

What is this file? Is it something created before the build runs? Can I build from the GUI? I don't understand if build.cmd should be used instead of the VS GUI. I'm using VS 2015. I downloaded the zip and tried to generate the project.
Thanks.

**Comment by cvalde** [comment]
Yes, I read the developers guide but finally took a simpler route. Ignored the missing SoliutionInfo.cs and built anyway from the VS GUI. I'm not creating a package but only a basic application to be able to debug TextExtractor. I made some modifications to observe the effect in the output. Apparently it's not possible to suppress the output of the "meta" lines like this one
<meta name="date" content="2018-05-28T20:13:00Z"/>
but I made the example output xml instead of plain text. For me it's a good start, thanks.

**Comment by KevM** [comment]
Hmm the guide  should have helped you through the `SolutionInfo.cs` problem. ☹️
